CI/travis.linux.install.sh

```diff
@@ -10,8 +10,9 @@ YAJL_PATH="$(pkg-config --variable=libdir yajl)"
 echo "Setting search path to $YAJL_PATH"
 luarocks install --local lua-yajl YAJL_LIBDIR="${YAJL_PATH}"
 
-if [ "${TRAVIS_EVENT_TYPE}" = "cron" ]; then
-  # download coverity tool only for cron jobs
+if [ "${TRAVIS_EVENT_TYPE}" == "cron" ] && [ "${TRAVIS_OS_NAME}" = "linux" ] \
```

Comment:
This script should be called on linux only, so the condition is not necessary.

Reply:
Fixed.